style/java.md

```diff
@@ -0,0 +1,33 @@
+# Java
+
+At Khan Academy, we only use Java for Android and, as such, follow the [Android Code Style Guidelines for Contributors](https://source.android.com/source/code-style.html). Note that those rules themselves extend the [standard Java language style guide as of 1999](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141855.html). The rest of this document describes additions and clarifications to those rules that we follow at Khan Academy.
+
+
+## Imports
+
+Import statements are divided into the following groups, in this order, with each group separated by a blank line:
```

Comment:
I would prefer to just do whatever Android Studio's default algorithm is, and not even document the order unless we expect people to be working on the project outside of Android Studio. But if we use a shared set of Android Studio settings and they encode the rules below, then I guess I would be fine with that as well. IMO, Java imports can and should be seen as "metadata for the compiler", and Java programmers should never have to read, write, or think about imports, except in the rare case when there are two classes with the same name.

But as long as a new person joining the project doesn't have to learn the import rules and doesn't have to do any additional setup just to get "organize imports" working, I'm fine with whatever we end up using.